src/main/java/Duke.java

```diff
@@ -1,10 +1,211 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+import duke.task.*;
+import java.io.IOException;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.File;
+import duke.dukeException.DukeParseException;
+import duke.Interpreter;
+import duke.Storage;
+import duke.TaskList;	
+import duke.Parser;
+import duke.commands.Command;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.layout.Region;
+import javafx.scene.control.Label;
+import duke.DialogBox;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;	
+import javafx.application.Platform;
+import duke.DukeResponse;
+
+public class Duke extends Application {
+	static final String separation = "_________________________________________________";
+	static final String greetingMessage = "Salue! Je suis Duke. \nWhat can I do for you?";
+	static final String pathToData = "data/storage.txt";
+	private Storage storage;
+	private TaskList taskList;
+	private Parser parser;
+	private ScrollPane scrollPane;
```

Comment:
may want to change indentation to 4 spaces instead to suit coding standard

Reply:
I think the final values should be named in capital letters.